### DISCUSSION TITLE
HOWTO: Using $flow and $vars in Custom JS Function nodes?

### DISCUSSION THREAD

**Discussion by dkindlund** [FlowiseAI/Flowise#1488] [Q&A] [answered]
For the life of me, I can seem to figure out how a Custom JS Function node can access $flow and $vars.  Can it be done?  If yes, then how?  Example code would be greatly appreciated!

**Comment by HenryHengZJ** [comment]
It's not possible yet. It's only available on custom tool function. We will bring this to custom JS function eventually

**Reply by HenryHengZJ** [reply, marked as answer]
PR - https://github.com/FlowiseAI/Flowise/pull/1558

**Reply by dkindlund** [reply]
Thank you very much, @HenryHengZJ !
